src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json

```diff
@@ -45,5 +45,8 @@
         "LocalAuthority"
       ]
     }
+  },
+  "Environment": {
+    "BaseUrl": "https://explore-education-statistics.service.gov.uk"
```

Comment:
Ah ok, didn't realise there was one already - the piece of code I was looking at at the time used a hardcoded domain so I assumed there wasn't a class configured yet.

Maybe these options are configured differently than I expected, but appsettings.json typically applies to production so it's expected to find a production URL here, I just didn't add similar values to appsettings.Development.json (etc.) just in case it wasn't wanted.

Reply:
`appsettings.json` is just the default settings file for keys not overridden elsewhere in configuration sources that take a higher precedence like environment variables and `appsettings.Development.json`.

We should probably have default or empty values for all configuration params  like `PublicAppUrl: ""`in `appsettings.json` so that it's obvious what the total set of configuration keys the application expects is.

`PublicAppUrl` is in `appsettings.Development.json` for our local environment and defined as an environment variable in the infra config for all Azure environments.

If you still need the base URL then I would defer to using `PublicAppUrl` or at least do some refactoring so we don't end up with two variables that are the same thing. For reference, [ConfigurationController.cs](https://github.com/dfe-analytical-services/explore-education-statistics/blob/88b29f9798dbfcc47b2063b4fbc148dd435dd22c/src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ConfigurationController.cs) is using it and note that the were recent changes to it in https://github.com/dfe-analytical-services/explore-education-statistics/pull/4575.